scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/SwaggerConfig.java

```diff
@@ -0,0 +1,55 @@
+package ca.on.oicr.gsi.runscanner.scanner;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger.web.*;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+  String projectName = "Run Scanner";
+
+  @Value("${project.version}")
+  String projectVersion;
+
+  @Bean
+  public Docket api() {
+    return new Docket(DocumentationType.SWAGGER_2)
+        .select()
+        .apis(RequestHandlerSelectors.any())
+        .paths(PathSelectors.any())
+        .build()
+        .apiInfo(metaData());
```

Comment:
Swagger's default responses include unauthorized and forbidden codes, which can never actually happen with Run Scanner since there is no authorization. I think this is the line in Pinery's Swagger config that removes all of the default responses for `GET` requests: https://github.com/oicr-gsi/pinery/blob/master/pinery-ws/src/main/java/ca/on/oicr/pinery/ws/component/SwaggerConfig.java#L37

Should change the default responses for `POST` as well, as there won't be unauthorized/forbidden responses, and the regular response for `/runs/progressive` should actually be 200 (OK) rather than 201 (Created)